src/styles/main.scss

```diff
@@ -0,0 +1,76 @@
+@import "./mixins";
+@import "./variables";
+@import "./keyframes";
+@import "./reset.scss";
+
+.content {
+  @include flexCenter;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  background-image: url(../images/background.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.clock {
+  &__hands {
+    background-image: url(../images/clock.png);
+    background-size: $clock-size $clock-size;
+    background-color: #ebece0;
+  }
+}
+
+.hands {
+  position: relative;
+  @include flexCenter;
+  width: $clock-size;
+  height: $clock-size;
+  border: 25px solid rgb(253,77,29);
+  border-radius: 50%;
+  box-shadow: $clock-shadow;
+
+  &__hour {
+    @include sharpeningNposition;
+    top: calc(50% - 150px);
+    height: 150px;
+    width: 12px;
+    background-color: $black-color;
+    box-shadow: $box-shadow;
+    animation: transform 43200s infinite linear;
+  }
+
+  &__minute {
+    @include sharpeningNposition;
+    top: calc(50% - 200px);
+    height: 200px;
+    width: 8px;
+    background-color: $black-color;
+    box-shadow: $box-shadow;
+    animation: rotate 3600s infinite linear;
+  }
+
+  &__second {
+    @include sharpeningNposition;
+    z-index: 1;
+    top: calc(50% - 175px);
+    height: 175px;
+    width: 4px;
+    background-color: #222831;
+    box-shadow: $box-shadow;
+    animation: rotate 60s infinite linear;
+  }
+
+  &__dot {
+    @include flexCenter;
+    position: relative;
+    z-index: 3;
+    top: calc(50% - 13px);
+    height: 26px;
+    width: 26px;
+    border-radius: 50%;
+    background-color: rgb(0, 0, 0);
+    border: 2px solid black;
+    box-shadow: $box-shadow;
+  }
+}
```

Comment:
[BEM] - create a separate file per each BEM block styles that have the same name as the block